docs/plugins/plugin_resources.rst

```diff
@@ -1,6 +1,84 @@
-.. vale off
-
 Plugin resources
 ################
 
-.. vale on
\ No newline at end of file
+Mautic Plugins are installable packages which can extend Mautic feature or integrate it with another system. You can find more information about how to create a Mautic Plugin in the :xref:`Plugins developer documentation`. 
+
+You can find the Plugins in the right Admin menu.
+
+.. image:: images/plugins.png
+  :width: 800
+  :alt: Screenshot of Plugins settings
+
+.. vale off
+
+Install Plugins
+***************
+
+.. vale on
+
+If you are on a freshly installed Mautic instance, there is a chance that you don't have the default Plugins installed yet. Click on the *Install/Upgrade Plugins* button in the top right corner and all the Plugins should appear.
+
+.. image:: images/install-plugins.png
+  :width: 800
+  :alt: Screenshot of install Plugins
+
+If you are trying to install a new Plugin that didn't come with the original Mautic installation files and it doesn't appear after you click ``Install/Upgrade plugins``, clear your Mautic cache and try the *Install/Upgrade Plugins* button again.
```

Comment:
Might we link to docs on how to clear your cache? I think we must have it in the documentation somewhere?

Reply:
Yes, troubleshooting but we didn't add the section in the new docs but I will add it as a section here

Reply:
Can we add a link?

Reply:
@favour-chibueze Adding as a new issue so we can merge this PR as it's blocking.